spec/models/petition_spec.rb

```diff
@@ -52,6 +52,30 @@
     it { is_expected.to validate_presence_of(:department).with_message(/must be completed/) }
     it { is_expected.to validate_presence_of(:creator_signature).with_message(/must be completed/) }
 
+    context "sponsor email validations" do
+
+      it 'is valid with 5 sponsor emails' do
+        sponsor_emails = ['test1@test.com', 'test2@test.com', 'test3@test.com', 'test4@test.com', 'test5@test.com']
+        expect(FactoryGirl.build(:petition, sponsor_emails: sponsor_emails)).to be_valid
+      end
+
+      it 'is not valid with less than 5 sponsor emails' do
```

Comment:
It is valid, it will just take the fist 20 of the sponsor emails and discard the rest.